solardatatools/algorithms/clipping.py

```diff
@@ -370,8 +370,8 @@ def calculate_cdf(self, data):
         y_rs = f(x_rs)
         return x_rs, y_rs
 
-    def make_problem(self, y, weight=1e1):
-        out = make_l2_l1d2_constrained(y, weight=weight)
+    def make_problem(self, y, weight=5, solver=None):
+        out = make_l2_l1d2_constrained(y, w1=weight, solver=solver)
```

Comment:
if it truly is not too much of a hassle, let's make this function align with the rest of the SD subroutines, i.e., return a result not a problem instance